src/is-my-room/index.ts

```diff
@@ -0,0 +1,9 @@
+export function isMyRoom(roomName: string): boolean{
+  return !!(
```

Comment:
I do understand how this works, but this is quite a complex statement.
It might be a good idea to work with early returns instead in order to increase readability.